### PR TITLE
disable clang-format for code that is unaligned on purpose

### PR DESCRIPTION
**Pull request by passing**
as discussed in https://github.com/jamulussoftware/jamulus/pull/1127

### CODE REVIEW

File: src/protocol.cpp [hunk 1]
```diff
@@ -2879,7 +2879,9 @@ bool CProtocol::ParseMessageFrame ( const CVector<uint8_t>& vecbyData,
 
     // Extract actual data -----------------------------------------------------
 
+// clang-format off
 // TODO this memory allocation is done in the real time thread but should be
+// clang-format on
```
Comment (hoffie) [inline src/protocol.cpp:2884]: Could be moved that to the end of the comment block for better readability?

**Review by hoffie — COMMENTED** [review]
Thanks, looks good to me! :)
One minor nit.

I did not spot any additional places via `grep '^//.*TODO' src/* -r5`, so it seems complete to me. Anything which might still have missed will hopefully show up in the formatting PR. :)



For context: As we agreed to keep such intentionally non-aligned comments it is important to disable clang-format in these places. Otherwise, clang-format would "fix" the alignment which would lead to a loss of information when the first complete clang-format run is performed (#1127).

File: src/protocol.cpp (hunk 1 above)
Comment (passing) [reply src/protocol.cpp]: ah, missed that one... now it's fixed

**Review by hoffie — APPROVED** [review]
Thanks :)

**Comment by passing** [comment]
closing as included in https://github.com/jamulussoftware/jamulus/pull/1127

**Pull request closed without merging**